### PR TITLE
Coworker create Searchkit over view of queue status

### PR DESCRIPTION
**Pull request by Damilare1**
Overview
----------------------------------------
This PR adds a Formbuilder page and SearchKit form that helps administrators with Queue privileges to view the status of Civi Queues.

Issue: https://lab.civicrm.org/dev/core/-/issues/4815

Before
----------------------------------------
The queue status can only be viewed using a SQL query

After
----------------------------------------
The status for all the queues can be viewed conveniently on the browser in `https://{CIVI_DOMAIN}/civicrm/queues`

Technical Details
----------------------------------------
The SearchKit and Formbuilder Afforms were added to the CiviCRM admin UI extension, hence this requires the installation of the extension to work.

Screenshot 2023-11-28 at 21 32 50

Comments
----------------------------------------
_Anything else you would like the reviewer to note_

### CODE REVIEW

**Comment by eileenmcnaughton** [comment]
test fail looks like an intermittant fail - test this please

CRM_Core_BAO_CustomFieldTest::testFileDisplayValueNoDescription
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-'              <a href="/index.php?q=civicrm/file&amp;reset=1&amp;id=1&amp;eid=3&amp;fcs=ac2331f78b7ee4a85825baa9d538456b360d6d2d8042fac562335aaa5ecb0eb4_1701211092_168" title="test_file.txt">\n
+'              <a href="/index.php?q=civicrm/file&amp;reset=1&amp;id=1&amp;eid=3&amp;fcs=ac2331f78b7ee4a85825baa9d538456b360d6d2d8042fac562335aaa5ecb0eb4_1701211093_168" title="test_file.txt">\n
                 <i class="crm-i fa-paperclip" aria-hidden="true"></i>\n
               </a>'

File: ext/civicrm_admin_ui/managed/SavedSearch_Queues.mgd.php [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  [
+    'name' => 'SavedSearch_Queues_SearchDisplay_Queues_Table_1',
+    'entity' => 'SearchDisplay',
+    'cleanup' => 'always',
+    'update' => 'unmodified',
+    'params' => [
+      'version' => 4,
+      'values' => [
+        'name' => 'Queues_Table_1',
+        'label' => E::ts('Queues Table 1'),
```
Comment (colemanw) [inline ext/civicrm_admin_ui/managed/SavedSearch_Queues.mgd.php:46]: No need to make translators deal with this.
```suggestion
        'label' => E::ts('Queues'),
```

**Comment by colemanw** [comment]
Thanks for this @Damilare1 
Sorry to present a moving target but we've recently switched to using `.aff.php` files instead of `.aff.json`. So ideally `afsearchCiviCRMQueues.aff.json` would be renamed and reformatted as php along the lines of:
https://github.com/civicrm/civicrm-core/blob/c602ff8431cb59ef057033769264117d5d1a8a71/ext/civicrm_admin_ui/ang/afsearchAdminContactTypes.aff.php#L1-L5

**Comment by eileenmcnaughton** [comment]
@colemanw the php version works on 5.68?

**Comment by colemanw** [comment]
@eileenmcnaughton yes, and also 5.67. (but this PR is for `master` so I'm not sure why that matters)

**Comment by eileenmcnaughton** [comment]
@colemanw cos we are deploying it on 5.68 :-)

**Comment by Damilare1** [comment]
> Thanks for this @Damilare1 Sorry to present a moving target but we've recently switched to using `.aff.php` files instead of `.aff.json`. So ideally `afsearchCiviCRMQueues.aff.json` would be renamed and reformatted as php along the lines of:
> 
> https://github.com/civicrm/civicrm-core/blob/c602ff8431cb59ef057033769264117d5d1a8a71/ext/civicrm_admin_ui/ang/afsearchAdminContactTypes.aff.php#L1-L5

That's insteresting, done!

File: ext/civicrm_admin_ui/ang/afsearchCiviCRMQueues.aff.php [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?php
+use CRM_CivicrmAdminUi_ExtensionUtil as E;
+
+return [
+  'type' => 'search',
+  'title' => E::ts('CiviCRM Queues'),
+  'icon' => 'fa-list-alt',
+  'server_route' => 'civicrm/queues',
```
Comment (colemanw) [inline ext/civicrm_admin_ui/ang/afsearchCiviCRMQueues.aff.php:8]: I'm happy to merge this PR. My only outstanding question is should the path perhaps be:
```suggestion
  'server_route' => 'civicrm/admin/queue',
```
What do you think @eileenmcnaughton ?

**Pull request merged**